aws/resource_aws_rds_cluster_instance.go

```diff
@@ -246,7 +246,7 @@ func resourceAwsRDSClusterInstanceCreate(d *schema.ResourceData, meta interface{
 
 	// reuse db_instance refresh func
 	stateConf := &resource.StateChangeConf{
-		Pending:    []string{"creating", "backing-up", "modifying", "configuring-enhanced-monitoring"},
+		Pending:    []string{"creating", "backing-up", "modifying", "configuring-enhanced-monitoring", "maintenance", "rebooting", "renaming", "resetting-master-credentials", "starting", "upgrading"},
```

Comment:
Really a nitpick, but do you mind line-wrapping this? i.e. put all the new pending states to the next line? I'll be easier to read that way.